_posts/2017-04-30-React-Native-for-iOS-devs.markdown

```diff
@@ -0,0 +1,671 @@
+
+Our app release cadence is still about a month long, moving to React Native hasn't changed that. We've automated the entire process, so it's a cultural artifact rather than technical. So a day or two for the App Store review is fine, ideally our betas should be getting longer than that for testing.
+
+## Doing it right per platform 
+
+I've used the term cross-platform quite flippantly in this article to describe code that can run on many platforms. We've tried thinking of it as React Native gives us the ability to think in cross-platform terms. We now have most of our app's root view controllers in React Native, which means we could make a pretty simple Android app re-using that code with a pretty small amount of effort.
+
+We don't though. The apps we make *need* to fit the platform and feel like the best experience you can get, when you're potentially buying a [> $100,000 artwork][artwork-search]. We'd need an Android engineer with a lot of experience to work on the app, they'd need to be able to work on React Native for Android when we hit roadblocks and to provide useful context on how the app should feel. _BTW, if this is you - get in touch (orta@artsymail.com) ;)_. 
+
+We already have some cross-platform code to get us started - but it won't feel like the app we want to create until there is native code to take that React and make it shine.
+
+So, where does the cross-platform aspect come from? For most users of React Native, native development has _always_ been unavailable to them, and now it's not, because React Native has lowered the barrier to entry. This means many people are aiming to make cross-platform React Native apps that are entirely in JavaScript ( see [Create React Native App][crna]. ) They are just trying to get _something_ shipped, which could be different from what *you the reader* are probably doing. As a native dev, you're probably more interested in making something really fit the platform and shine. Lots of probablys, yes, but it fits the questions we're being asked.
+
+Both aspects of this are reasonable. React Native will let you do both. While React Native encourages you to think entirely at the V in MVC only, we still structure our React Native into `UIViewController`s and still support unique iOS features via native code.
+
+One place where this difference in perspective shows up is with navigation. In iOS it's pretty straight forwards: use `UINavigationController`. For React Native, it's actually _really tricky_, check out [this issue where I write up all the trade-offs][nav-options] for the current set of APIs. Some of this is just a statement that figuring out the right abstraction is hard here, the other is that a lot of big apps are also at the same size as ours with ~2 years of React Native adoption.
+
+## Create React Native App
```

Comment:
You are discussing it here, so it makes even more sense to bring it up in the example above, no?